### ISSUE TITLE
after failed connection attempt, does not "release" connection attempt/details for new attempt

### ISSUE THREAD

**Issue by zippydan** [conversejs/converse-desktop#9]
Using macOS version on BigSur

If I put incorrect credentials in to login, the "Connecting" notice just keeps spinning forever.

In my case, I put the incorrect URL for the Connection Manager.

Since there is no convenient cancel button, I had to use the "Force Logout" option from the application menu.

This returns me to the login window, but using the "Debugging" window and the "Network" pane, I could see that converse.js _continues to attempt to connect to the previously inputted Connection Manager even though I have "canceled" the connection attempt._

Even after I input a new Connection Manager URL and new login credentials and start a new login attempt, converse.js continues unsuccessfully trying to establish a connection using the first attempt with the first inputted credentials, and never stops, effectively ignoring the newly-inputted credentials.

The newly-inputted credentials are not _entirely_ ignored.  If I quit the converse.js app completely, and then relaunch it, it will then automatically attempt to connect with the last set of credentials I inputted before closing the app.

In short:

1. converse.js should have a method to fully cancel an attempted connection - currently it does not
2. converse.js should not require me to close and reopen the app to attempt a new connection with new credentials - currently it does

**Comment by lennylen** [comment]
+1

Describe the bug
When I enter in the BOSH address, username@xmpp.domain.com and password, nothing happens when i click add account. All the fields reset. If i restart the app it just shows a blank display.

To Reproduce
Steps to reproduce the behavior:

Enter details on first launch
Nothing happens, so restart the app
App is now restarted and shows blank screen
If i go to debug mode I see one error in error console:
index.html:1 Uncaught (in promise) Error: The name org.freedesktop.secrets was not provided by any .service files

Expected behavior
To login and chat.


Environment (please complete the following information):
Desktop App. Debian 10 Stable

Additional context
none

**Comment by lennylen** [comment]
Workaround for now on Ubuntu/Debian:
rm ~/.config/@converse/desktop/Settings 


> Using macOS version on BigSur
> 
> If I put incorrect credentials in to login, the "Connecting" notice just keeps spinning forever.
> 
> In my case, I put the incorrect URL for the Connection Manager.
> 
> Since there is no convenient cancel button, I had to use the "Force Logout" option from the application menu.
> 
> This returns me to the login window, but using the "Debugging" window and the "Network" pane, I could see that converse.js _continues to attempt to connect to the previously inputted Connection Manager even though I have "canceled" the connection attempt._
> 
> Even after I input a new Connection Manager URL and new login credentials and start a new login attempt, converse.js continues unsuccessfully trying to establish a connection using the first attempt with the first inputted credentials, and never stops, effectively ignoring the newly-inputted credentials.
> 
> The newly-inputted credentials are not _entirely_ ignored. If I quit the converse.js app completely, and then relaunch it, it will then automatically attempt to connect with the last set of credentials I inputted before closing the app.
> 
> In short:
> 
> 1. converse.js should have a method to fully cancel an attempted connection - currently it does not
> 2. converse.js should not require me to close and reopen the app to attempt a new connection with new credentials - currently it does

**Comment by lennylen** [comment]
I deleted the Settings file and heres what happens:

Attempt to login again with all 3 credentials
Click Add Account
Form resets

Debug log (error console):
`Uncaught (in promise) Error: The name org.freedesktop.secrets was not provided by any .service files
index.html:1 Uncaught (in promise) `

`Error: The name org.freedesktop.secrets was not provided by any .service files`

Screenshot:
![Screenshot_20210915_003226](https://user-images.githubusercontent.com/15621636/133371031-b3fdd290-cb71-4c77-9fed-8efc3a916a24.png)
